README.md

```diff
@@ -85,33 +85,40 @@ propertiesPanel.attachTo('#other-properties');
 
 ### Use with Camunda properties
 
-In order to be able to edit [Camunda](https://camunda.com) related properties, use the [camunda properties provider](https://github.com/bpmn-io/bpmn-js-properties-panel/tree/master/lib/provider/camunda).
+In order to be able to edit [Camunda](https://camunda.com) related properties, use the [camunda platform properties provider](https://github.com/bpmn-io/bpmn-js-properties-panel/tree/master/src/provider/camunda-platform).
 In addition, you need to define the `camunda` namespace via [camunda-bpmn-moddle](https://github.com/camunda/camunda-bpmn-moddle).
 
 ```javascript
-var BpmnJS = require('bpmn-js/lib/Modeler'),
-    propertiesPanelModule = require('bpmn-js-properties-panel'),
-    // use Camunda properties provider
-    propertiesProviderModule = require('bpmn-js-properties-panel/lib/provider/camunda');
+import BpmnModeler from 'bpmn-js/lib/Modeler';
+import {
+  BpmnPropertiesPanelModule,
+  BpmnPropertiesProviderModule,
+  
+  // use Camunda Platform properties provider
+  CamundaPlatformPropertiesProviderModule
+} from 'bpmn-js-properties-panel';
 
-// a descriptor that defines Camunda related BPMN 2.0 XML extensions
-var camundaModdleDescriptor = require('camunda-bpmn-moddle/resources/camunda');
+// use Camunda BPMN Moddle extension
+import CamundaExtensionModule from 'camunda-bpmn-moddle/lib',
```

Comment:
We should export the extension from the root 🙄  But we don't, so the code change makes sense.